### PR TITLE
1.0.0rc2

### PR DESCRIPTION
**Pull request by ogrisel**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-  sha256: d44e568114edbd341e5b5e901f7b43c91ef55850da2cf092ee7b018230c0e1f9
+  sha256: cf35a3dbbdba967b3851dce2b41ba0fc668ab365be47a2ed3e8353fdc035446e
+  patches:
+   - patches/0001-ignore-errors-when-tearing-down-modules.patch
 
 build:
-  number: 0
-  skip: true  # [py2k]
+  number: 2
+  skip: true # [py<37]
```
Comment (ogrisel) [inline recipe/meta.yaml:15]:
```suggestion
  skip: true  # [py<37]
```

**Comment by ogrisel** [comment]
@conda-forge-admin, please rerender

**Comment by ogrisel** [comment]
@adrinjalali it seems ready to merge.

**Comment by adrinjalali** [comment]
We need to document this in our release guidelines. I also don't like that there are practically two branches and we need to keep them in sync

**Comment by adrinjalali** [comment]
thanks a lot for doing it @ogrisel

**Comment by ogrisel** [comment]
> I also don't like that there are practically two branches and we need to keep them in sync

I am not sure what we can do about it.

**Comment by jakirkham** [comment]
Thanks for doing all of this Olivier! 😄

**Pull request merged**